enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -985,7 +1015,7 @@ func (c *podmanCommandContainer) State(ctx context.Context) (*rnpb.ContainerStat
 
 func runPodman(ctx context.Context, subCommand string, stdio *commandutil.Stdio, args ...string) *interfaces.CommandResult {
 	command := []string{"podman"}
-	if *transientStore {
+	if podmanVersion != nil && !podmanVersion.LessThan(transientStoreMinVersion) {
 		// Use transient store to reduce contention.
 		// See https://github.com/containers/podman/issues/19824
 		command = append(command, "--transient-store")
```

Comment:
wdyt about adding a logged-once-per-executor message when the podman version is too old (or not found) for this flag?

Reply:
It should always be set, except during the initial call to `runPodman` to get the version. I switched to just use `exec.Command().CombinedOutput()` though which seems cleaner.

Reply:
oh, re. it being too old, I will add that.

Reply:
done